src/pages/home.js

```diff
@@ -0,0 +1,130 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import logo from '../art/logo.svg';
+import './home.css';
+
+const Home = (props) => {
+  const {
+    games, pcGames, consoleGames, mobileGames, allGames, allDLC, allEarly,
+  } = props;
+
+  return (
+    <>
```

Comment:
- [ ] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name.